repose-aggregator/artifacts/valve/build.gradle

```diff
@@ -161,6 +163,13 @@ ospackage {
         permissionGroup 'repose'
         fileMode 0640
         fileType CONFIG | NOREPLACE
+
+        //INFO: We'd like to use the more standard gradle 'expand( docsHome: docsHome)',
+        // however it chokes pretty hard on the regexs we sprinkle into our xml specifically the `\`.
+        // We can update our config file examples to not have that in the regex,
+        // however some of the schemas we've imported also have them, and we shouldn't mess with those.
+        // Instead I've elected to use ant based filtering.
```

Comment:
Great comment. Answers the question I had bubbling while looking at the files above perfectly.